rosstatic/main.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# TODO: add custom header, footer
+# TODO: symlinks or copy param
+
+import subprocess
+import os
+import shutil
+import rospy
+
+rospy.init_node('rosstatic')
+
+www = os.path.dirname(os.path.realpath(__file__)) + '/www'
+index_file = rospy.get_param('~index', None)
+
+shutil.rmtree(www, ignore_errors=True)  # reset www directory content
+os.mkdir(www)
+
+lines = subprocess.check_output(['rospack', 'list']).split('\n')
```

Comment:
I believe there's a better, more ROS-like way to do that: https://docs.ros.org/independent/api/catkin_pkg/catkin_pkg.html#module-catkin_pkg.packages

Reply:
Well, that's the shortest way I could find.

Reply:
Okay, what about `rospkg`? https://wiki.ros.org/Packages#Client_Library_Support

Reply:
Done.